lib/top_hangman/round.rb

```diff
@@ -1,9 +1,39 @@
+require_relative 'render'
+require_relative 'guess'
+
 module TopHangman
   class Round
     attr_reader :word
 
-    def initialize(word)
+    def initialize(word, render)
       @word = word
+      @guess = Guess.new(@word.current_word, render)
+      @render = render
+      @render.setup_guess(@guess)
+    end
```

Comment:
instead of receiving the `word` as argument, instantiate it by calling `@word = Word.from_file`